uiowa/uiowa
# Default LB off-canvas dialog to 500px and style drag handle.

Resolves #1366 

Attempting this, I definitely came around on #1927 - in fact I went further and just hid it entirely on layout builder forms because it **does not** show node information at all. Maybe we should do this for all admin paths so it only displays on the front end?

I tried to set the dialog width using the Drupal API but that was proving difficult: https://www.drupal.org/project/drupal/issues/2951547 Eventually went with JS since the existing inline styles can't be removed from the element.

I ripped off the lb_ux tray exactly. If someone has style ideas for this, I would say go for it in this PR.

### Testing
- Check out this branch.
- Edit a layout on a collegiate site.
- See the wider tray and drag handle.
- See that the form elements do not have extra space on the right after making the tray more narrow.
- Test multiple browsers, viewport widths, etc.
